test/Microsoft.TestPlatform.CommunicationUtilities.PlatformTests/SocketServerTests.cs

```diff
@@ -35,9 +35,11 @@ public void Dispose()
 #if NETFRAMEWORK
             // tcpClient.Close() calls tcpClient.Dispose().
             this.tcpClient?.Close();
+            GC.SuppressFinalize(this);
 #else
             // tcpClient.Close() not available for netcoreapp1.0
             this.tcpClient?.Dispose();
+            GC.SuppressFinalize(this);
```

Comment:
nit: We can remove this call and add it outside of `#if` preprocessor directive, just after `#endif`.

Reply:
resolved added GC.SuppressFinalize(this); after the #endif